### ISSUE TITLE
Unable to get property 'kind' of undefined or null reference

### ISSUE THREAD

**Issue by ThinhTT84** [bobril/bbcore#7]
While using bb2 with webbobwai, I have below exception:

```
Fatal Error: JavaScriptEngineSwitcher.Core.JsRuntimeException: TypeError: Unable to get property 'kind' of undefined or null reference
   at evalNode (Lib.TSCompiler.bbtsc.js:279:5)
   at evalNode (Lib.TSCompiler.bbtsc.js:419:21)
   at evalNode (Lib.TSCompiler.bbtsc.js:411:17)
   at evalNode (Lib.TSCompiler.bbtsc.js:411:17)
   at visit (Lib.TSCompiler.bbtsc.js:513:21)
   at visitNode (typescript.js:13263:9)
   at forEachChild (typescript.js:13338:17)
   at visit (Lib.TSCompiler.bbtsc.js:620:9)
   at visitNodes (typescript.js:13272:17)
   at forEachChild (typescript.js:13421:17) ---> JavaScriptEngineSwitcher.ChakraCore.JsRt.JsScriptException: Script threw an exception.
   at JavaScriptEngineSwitcher.ChakraCore.JsRt.JsErrorHelpers.ThrowIfError(JsErrorCode error)
   at JavaScriptEngineSwitcher.ChakraCore.JsRt.JsValue.CallFunction(JsValue[] arguments)
   at JavaScriptEngineSwitcher.ChakraCore.ChakraCoreJsEngine.<>c__DisplayClass42_0.<InnerCallFunction>b__0()
   --- End of inner exception stack trace ---
   at JavaScriptEngineSwitcher.ChakraCore.ChakraCoreJsEngine.<>c__DisplayClass42_0.<InnerCallFunction>b__0()
   at JavaScriptEngineSwitcher.ChakraCore.ScriptDispatcher.<>c__DisplayClass12_0`1.<Invoke>b__0()
   at JavaScriptEngineSwitcher.ChakraCore.ScriptDispatcher.StartThread()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at JavaScriptEngineSwitcher.ChakraCore.ScriptDispatcher.InnnerInvoke(Func`1 del)
   at JavaScriptEngineSwitcher.ChakraCore.ScriptDispatcher.Invoke[T](Func`1 func)
   at Lib.TSCompiler.TsCompiler.GatherSourceInfo()
   at Lib.TSCompiler.TSProject.Build(BuildCtx buildCtx)
   at Lib.Composition.Composition.<InitInteractiveMode>b__38_1()
Build done in 44.8 with 1 error and no warnings and has no files
```

**Comment by Bobris** [comment]
I will need access to your webbobwai content.

**Comment by ThinhTT84** [comment]
fixed in 0.18.0
